src/core/utils/querybuilder.typeorm.ts

```diff
@@ -0,0 +1,81 @@
+import { BaseEntity, Brackets, SelectQueryBuilder } from 'typeorm';
+export const ExtraQueryBuilder = {
+  async paginateBy<T extends BaseEntity>(

```

Comment:
Update this function to return more value:
page,
pageSize,
totalPages,
totalDocs {Number} - Total number of documents in collection that match a query.
prevPage {Number} - Previous page number if available or NULL
nextPage {Number} - Next page number if available or NULL